src/explore-education-statistics-common/src/components/form/__tests__/FormComboBox.test.tsx

```diff
@@ -1,804 +1,812 @@
 import React from 'react';
-import { fireEvent, render } from '@testing-library/react';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import noop from 'lodash/noop';
+import renderer from 'react-test-renderer';
+import userEvent from '@testing-library/user-event';
 import FormComboBox from '../FormComboBox';
 
 describe('FormComboBox', () => {
   test('renders with no options by default', () => {
-    const { container } = render(
+    render(
       <FormComboBox
         id="test-combobox"
         inputLabel="Choose option"
-        onInputChange={() => {}}
-        onSelect={() => {}}
+        onInputChange={noop}
+        onSelect={noop}
         options={['Option 1', 'Option 2', 'Option 3']}
       />,
     );
 
-    const options = container.querySelectorAll('[role="option"]');
+    const options = screen.queryAllByRole('option');
 
     expect(options).toHaveLength(0);
-    expect(container.innerHTML).toMatchSnapshot();
+
+    const tree = renderer
+      .create(
+        <FormComboBox
+          id="test-combobox"
+          inputLabel="Choose option"
+          onInputChange={noop}
+          onSelect={noop}
+          options={['Option 1', 'Option 2', 'Option 3']}
+        />,
+      )
+      .toJSON();
```

Comment:
Why do this? It's okay to use `container` from the `render` above to test that the HTML matches the snapshot